indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -1,74 +1,70 @@
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
 
-    $('.event-service-row > .trigger').on('click', function() {
-        var toggler = $(this);
-        toggler.siblings('.event-service-details').slideToggle({
-            start: function() {
-                toggler.toggleClass('icon-expand icon-collapse');
-            },
-            duration: 'fast'
-        });
-    });
+(function(global) {
+    'use strict';
 
-    $('.event-service-row-toggle').on('click', function(e) {
-        e.preventDefault();
-        var toggler = $(this);
-        var toggler_button = $(this).parent().siblings('.trigger');
-        toggler.parent().siblings('.event-service-details').slideToggle({
-            start: function() {
-                toggler_button.toggleClass('icon-expand icon-collapse');
-            },
-            duration: 'fast'
+    global.setupEventDisplay = function setupEventDisplay() {
+        $('.manage-button').dropdown();
+        $('.js-go-to-day').dropdown({
+            always_listen: true
+        }).find('li a').on('menu_select', function() {
+            window.location = $(this).attr('href');
+            return false;
         });
-    });
 
-    var gradientLayer = $('.participant-list-wrapper > .gradient-layer');
-    var heightControl = $('<div>', {'class': 'height-control'});
-    gradientLayer.append(heightControl);
-    var threeRowsHeight = heightControl.height();
-    $('.participant-list-wrapper').toggleClass('collapsible collapsed',
-        $('.participant-list').height() > threeRowsHeight);
-    var initialHeight = $('.participant-list-wrapper').height();
-    heightControl.remove();
-
-    $('.participant-list-wrapper > .trigger, .participant-list-wrapper > .gradient-layer').on('click', function() {
-        var toggler = $('.participant-list-wrapper > .trigger');
-        var participantList = toggler.siblings('.participant-list');
-        var wrapper = participantList.parent();
-        if (wrapper.hasClass('collapsed')) {
-            var newHeight = participantList.height();
-            participantList.height(initialHeight);
-            wrapper.find('.gradient-layer').fadeOut();
-            wrapper.removeClass('collapsed');
-            wrapper.animate({
-                height: newHeight
-            }, {
-                duration: 'fast',
-                start: function() {
-                    toggler.addClass('icon-collapse').removeClass('icon-expand');
-                },
-                complete: function() {
-                    participantList.height(newHeight);
+        $('.notes-editor').on('click', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            ajaxDialog({
```

Comment:
I see three very similar snippets for `ajaxDialog` in this file. Maybe we could adapt `data-ajax-dialog` a accept `confirmCloseUnsaved` and to reload on close instead of updating the html.